src/algorithms/rsa.rs

```diff
@@ -114,7 +114,7 @@ impl RSAKeyPair {
         self.rsa_sk
             .to_pkcs8_der()
             .map_err(Into::into)
-            .map(|x| x.as_ref().to_vec())
+            .map(|x| std::mem::take(x.to_bytes().as_mut()))
```

Comment:
RSA now returns a `Zeroizing<Vec<u8>>` and I found no easier way to unwrap it.

Reply:
Congrats for figuring out that magic :)